query_server/sqllogicaltests/cases/sys_table/usage_schema/http_metrics.slt

```diff
@@ -1,7 +1,24 @@
+statement ok
+--#DATABASE=http_stream_select
```

Comment:
use another database name, and drop it when finish

Reply:
done,